docs/Maintainer-Guidelines.md

```diff
@@ -7,6 +7,8 @@ definitely not a beginner’s guide.
 
 Maybe you were looking for the [Formula Cookbook](Formula-Cookbook.md)?
 
+This document is a work in progress. If you wish to change or discuss any of the below: open a PR to suggest a change.
```

Comment:
Re-reading this, calling this a "work in progress" is probably not _correct_. Rather, calling it a "living document" or "current practice" or "amendable standard" or something like that is more appropriate. The second sentence alludes to this but the first sentence doesn't bear the _confidence_ that such a document deserves.